src/components/AggregateCriteria/AggregateCriteriaCard.tsx

```diff
@@ -0,0 +1,129 @@
+import { Card, Col, Row, Select, TextField } from '@folio/stripes/components';
+import React, { useMemo } from 'react';
+import { CriteriaAggregateType } from '../../types/CriteriaTypes';
+import { Field, useField } from 'react-final-form';
+import OperatorSelect from '../Criteria/OperatorSelect';
+import useMonetaryOnBlur from '../../hooks/useMonetaryOnBlur';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+export default function AggregateCriteriaCard() {
+  const selectedType = useField<CriteriaAggregateType>('aggregateFilter.type', {
+    subscription: { value: true },
+    format: (value) => value ?? CriteriaAggregateType.PASS,
+  }).input.value;
+
+  const monetaryOnBlur = useMonetaryOnBlur('aggregateFilter.amountDollars');
+  const intl = useIntl();
+
+  const criteriaOptions = useMemo(
+    () =>
+      [
+        {
+          label: intl.formatMessage({
+            id: 'ui-plugin-bursar-export.bursarExports.aggregate.filter.numAccounts',
+          }),
+          value: CriteriaAggregateType.NUM_ROWS,
+        },
+        {
+          label: intl.formatMessage({
+            id: 'ui-plugin-bursar-export.bursarExports.aggregate.filter.totalAmount',
+          }),
+          value: CriteriaAggregateType.TOTAL_AMOUNT,
+        },
+      ].sort((a, b) => a.label.localeCompare(b.label)),
+    [intl]
+  );
+
+  return (
+    <Card
+      headerStart={
+        <FormattedMessage id="ui-plugin-bursar-export.bursarExports.aggregate.filter.header" />
+      }
+    >
+      <Row>
+        <Col xs={12} md={4}>
+          <Field
+            name="aggregateFilter.type"
+            defaultValue={CriteriaAggregateType.PASS}
+          >
+            {(fieldProps) => (
+              <Select<CriteriaAggregateType>
+                {...fieldProps}
+                fullWidth
+                marginBottom0
+                required
+                label={
+                  <FormattedMessage id="ui-plugin-bursar-export.bursarExports.aggregate.filter" />
+                }
+                dataOptions={[
+                  {
+                    label: intl.formatMessage({
+                      id: 'ui-plugin-bursar-export.bursarExports.aggregate.filter.none',
+                    }),
+                    value: CriteriaAggregateType.PASS,
+                  },
+                  ...criteriaOptions,
```

Comment:
what is the purpose of memo, when you recreate array? as an option you can add none option in memo after sorting